[2024-05-30T15:42:00.587+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T15:42:00.626+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_dag.scrap_netflix manual__2024-05-30T15:41:36.715394+00:00 [queued]>
[2024-05-30T15:42:00.634+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_dag.scrap_netflix manual__2024-05-30T15:41:36.715394+00:00 [queued]>
[2024-05-30T15:42:00.635+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-30T15:42:00.645+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): scrap_netflix> on 2024-05-30 15:41:36.715394+00:00
[2024-05-30T15:42:00.657+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=256) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-30T15:42:00.658+0000] {standard_task_runner.py:63} INFO - Started process 269 to run task
[2024-05-30T15:42:00.656+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_dag', 'scrap_netflix', 'manual__2024-05-30T15:41:36.715394+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpt081a0_r']
[2024-05-30T15:42:00.660+0000] {standard_task_runner.py:91} INFO - Job 116: Subtask scrap_netflix
[2024-05-30T15:42:00.728+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_dag.scrap_netflix manual__2024-05-30T15:41:36.715394+00:00 [running]> on host cf97ea5db7f3
[2024-05-30T15:42:00.849+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_dag' AIRFLOW_CTX_TASK_ID='scrap_netflix' AIRFLOW_CTX_EXECUTION_DATE='2024-05-30T15:41:36.715394+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-30T15:41:36.715394+00:00'
[2024-05-30T15:42:00.855+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T15:42:00.863+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T15:42:00.867+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/test.py", line 15, in download_netflix
    from src.netflix_downloader import download_netflix_data
ModuleNotFoundError: No module named 'src'
[2024-05-30T15:42:00.903+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=s3_dag, task_id=scrap_netflix, execution_date=20240530T154136, start_date=20240530T154200, end_date=20240530T154200
[2024-05-30T15:42:00.915+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 116 for task scrap_netflix (No module named 'src'; 269)
[2024-05-30T15:42:00.966+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-30T15:42:00.976+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-30T15:42:00.977+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
